synthetic.md

```diff
@@ -164,31 +165,41 @@ Please see the contact us section to let us know of further work
 
 ### Datasets 
 [The Simulacrum - healthdatainsight.org.uk](https://healthdatainsight.org.uk/project/the-simulacrum/)
+
 Fidelity = Medium
+
 Technique = Probabilistic Model - Chi-Squared
 
 The Simulacrum is a dataset that contains artificial patient-like cancer data to help researchers gain insights. The Simulacrum imitates some of the data held securely by the Public Health England’s National Cancer Registration and Analysis Service. The data is synthetic and does not contain any information about real patients. It is free to use and allows anyone who wants to use record-level cancer data to do so, safe in the knowledge that while the data 
```

Comment:
Issue raised about truncated sentence here - can this be fixed as part of this PR?
 [Issue 3](https://github.com/nhsx/AnalyticsUnit/issues/3)